examples/default/Makefile

```diff
@@ -35,9 +35,9 @@ USEMODULE += ps
 USEMODULE += saul_default
 
 BOARD_PROVIDES_NETIF := airfy-beacon cc2538dk fox iotlab-m3 iotlab-a8-m3 mulle \
-        microbit native nrf51dongle nrf52dk nrf6310 openmote-cc2538 pba-d-01-kw2x \
-        pca10000 pca10005 remote-pa remote-reva saml21-xpro samr21-xpro \
-        spark-core telosb yunjia-nrf51822 z1
+        microbit native nrf51dongle nrf52dk nrf6310 acd52832 openmote-cc2538 \
```

Comment:
this list is lexicographically sorted, hence `acd52832` should be in pole position 😄 though it might be based on NRFxy and hence related to nrf6310

Reply:
:+1: